js-foundations.js

```diff
@@ -2,25 +2,46 @@
 // Coding practical questions
 
 // 1. Write an arrow function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
-
-
+const divideBy3 = (num) => {
+    num = -7;
```

Comment:
Let's define and declare our variable outside of the function, then pass in that variable into the function when invoking it.